build.gradle

```diff
@@ -100,6 +100,9 @@ allprojects {
         maven {
             url "https://maven.research.rackspacecloud.com/content/groups/aggregate/"
         }
+        maven {
+            url 'https://build.shibboleth.net/nexus/content/repositories/releases'
```

Comment:
Was this where the opensaml stuff was? if so i think we can burn it with fire now , yes? Also, this explains so much.